nemo/collections/asr/models/aed_multitask_models.py

```diff
@@ -387,6 +387,8 @@ def transcribe(
         num_workers: int = 0,
         channel_selector: Optional[ChannelSelectorType] = None,
         augmentor: DictConfig = None,
+        text_field: str = "text",
```

Comment:
This isnt very good - adding such fields opens the way for dozens of other fields like src_text field and dest_text_field and pnc etc for all models even those that don't use it.

Better way of doing it is to hide them inside of the data classes of these models. If the user needs to override the default, they can instantiate an object of the dataclass and use override_config